frontend/src/pages/Dashboard/StudentDashboard.tsx

```diff
@@ -391,18 +391,35 @@ interface SessionCardParams {
   sessionNumber: number;
   image: string;
   isComplete: boolean;
+  studentId: string;
+  isStudentView: boolean;
 }
 
 const SessionCard: FunctionComponent<SessionCardParams> = ({
   sessionNumber,
   image,
   isComplete,
+  studentId,
+  isStudentView,
 }) => {
-  return (
+  const history = useHistory();
+  return isStudentView || !isComplete ? (
     <SessionBox isComplete={isComplete}>
       <SessionImage src={image} />
       <SessionNumber>session {sessionNumber}</SessionNumber>
     </SessionBox>
+  ) : (
+    <SessionBox
+      isComplete={isComplete}
+      onClick={() =>
+        history.push(
+          `/dashboard/${studentId}/session/${sessionNumber}`
+        )
+      }
+    >
+      <SessionImage src={image} />
+      <SessionNumber>session {sessionNumber}</SessionNumber>
+    </SessionBox>
```

Comment:
I think instead of having two different things here, we can just have one, and in the `onClick` check if it is the student view or not complete. Another comment is can we make the cursor `pointer` and the `opacity` 0.8 when you can click on it to show that you can click there?